interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -709,7 +709,9 @@ def test_setHP_highT(self):
         self.assertTrue(self.gas.T > self.gas.max_temp)
 
     def test_volume(self):
-        """ This phase should follow the ideal gas law """
+        """
```

Comment:
I keep forgetting to fix this. I have no idea why this gets printed in the first place, but am confused as to why just reformatting the docstring would change the behavior. Should it just be comment instead?